### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by aldemeery**
### Added

- Testing Evaluator.
- Testing Game.
- Testing Board.
- Testing Player.

### CODE REVIEW

File: test/Board.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  board.put('X', 1);
+  expect(board.get(1)).toBe('X');
+});
+
+test('Checkinf if a board is filled', () => {
+  const board = new Board();
+  expect(board.isFilled()).toBe(false);
+  board.board = new Array(9).fill('X');
+  expect(board.isFilled()).toBe(true);
+});
```
Comment (shshamim63) [inline test/Board.test.js:27]: [Optional]
You can place all the tests inside a group that is under Board, and before each of the tests, you had to create an instance of that class. I tried the following way, maybe you can give it a shot.
```js
describe('Board', () => {
  let board;
  beforeEach(() => {
    board = new Board();
  });

  test('Board construction', () => {
    expect(board.board).toStrictEqual(new Array(9).fill(''));
    expect(board.dimension).toBe(3);
  });

  test('Getting cells from board', () => {
    expect(board.get(1)).toBe('');
    board.board[1] = 'X';
    expect(board.get(1)).toBe('X');
  });

  test('Setting cells of board', () => {
    board.put('X', 1);
    expect(board.get(1)).toBe('X');
  });

  test('Checkinf if a board is filled', () => {
    expect(board.isFilled()).toBe(false);
    board.board = new Array(9).fill('X');
    expect(board.isFilled()).toBe(true);
  });
});
```

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## :recycle: Required Changes :recycle: 
___
Hi @aldemeery :wave: 
Nice work, you have tested all the possible scenarios that could happen while playing the game :clap: . But your Readme does not contain any info related to testing. Without a complete readme, I cannot approve this now. Please add it and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

File: test/Board.test.js (hunk 1 above)
Comment (aldemeery) [reply test/Board.test.js]: Resolved.

**Review by rOluochKe — APPROVED** [review]
Hello Team
### You got it right this time, well done :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next project.
- Happy coding!

**Pull request merged**